Lib/glyphsLib/classes.py

```diff
@@ -1747,6 +1759,87 @@ def _set_metric(self, metricname, value):
             self.metrics.append(GSMetricValue(position=0))
         self.metrics[metricIndex] = GSMetricValue(position=value)
 
+    @property
+    def alignmentZones(self):
+        # If there are values that are parsed from file format 2 or which are
+        # explicitly set return those
+        if self._alignmentZones is not None:
+            return self._alignmentZones
+
+        if len(self.metrics) == 0:
+            return []
+
+        zones = []
+        for index, fontMetric in enumerate(self.font.metrics):
+            # Ignore the "italic angle" "metric", it is not an alignmentZone
+            if fontMetric.type == "italic angle":
+                continue
+            metric = self.metrics[index]
+            # Ignore metric without overshoot, it is not an alignmentZone
+            if metric.overshoot == 0:
+                continue
```

Comment:
can you confirm that Glyphs.app v3 also does the same (ignore 'italic angle' metric an those with overshoot=0) when you get a GSFont.alignmentZones?
What's an example of such a metric without overshoot?

Reply:
Yes. This is what Glyphs is doing.